### DISCUSSION TITLE
Making apps stand-alone

### DISCUSSION THREAD

**Discussion by markus-li** [DustinBrett/daedalOS#133] [Feature Requests]
Have you thought about making apps within the codebase fully stand-alone so that they can easily be added and removed? Considering the effort you have put into this, I wouldn't be the least bit surprised if you have. I guess my question more is, how are you planning to accomplish this if you already have a plan for it? Looking at your codebase there's plenty of flexibility and I'm very much liking the general structure as is. Was looking at your code a bit today and was thinking about how to best make additions without touching the original project code. I didn't find a clean way yet, but that might just be me not seeing it.

**Comment by DustinBrett** [comment]
As far as adding apps, I have indeed been considering this (https://github.com/DustinBrett/daedalOS/discussions/37) and will have something by the end of the year hopefully that would allow people to write apps that could be put into daedalOS with a simple install or file copy. For the other direction of taking existing apps in daedalOS and having them live outside, I have not given that much thought at the moment. Perhaps a similar idea I will eventually be pursuing is a PWA and a full Electron style app, so that daedalOS itself can be ran on the desktop outside the browser.
